scheduler/src/cook/mesos/constraints.clj

```diff
@@ -82,7 +79,15 @@
     [this _ vm-attributes _]
     (job-constraint-evaluate this _ vm-attributes)))
 
-(defrecord gpu-host-constraint [job]
+(defn build-novel-host-constraint
+  "Constructs a novel-host-constraint"
```

Comment:
Should we also add that the constraint is meant to exclude any hosts the job previously ran on?